src/charm.py

```diff
@@ -205,6 +267,7 @@ def _config(self) -> dict:
             "root_password": "password",
             "security_key": "",
             "unit_ips": self._unit_ips,
+            "calling_unit_ip": str(self.model.get_binding(PEER).network.bind_address),
```

Comment:
Interesting. I will steal, I mean, leverage this for MySQL. (:

Reply:
Happy its useful! @marceloneppel also found it useful, perhaps we can add something to [unit](https://ops.readthedocs.io/en/latest/#ops.model.Unit) so we could do something like `self.unit.ip_address`. I'll try to remember to talk to @jnsgruk about this sometime this week